### PR TITLE
enable fips compliance on openshift operator

### PR DESCRIPTION
**Pull request by jkhelil**
# Changes
Enable FIPS compliance on openshift operator

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Run `make test lint` before submitting a PR
- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/operator/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

```release-note
NONE
```

### CODE REVIEW

**Review by jkandasa — APPROVED** [review]
cc @vdemeester

**Comment by jkhelil** [comment]
@vdemeester @savitaashture ptal

File: operatorhub/openshift/manifests/bases/openshift-pipelines-operator-rh.clusterserviceversion.template.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ metadata:
     description: Red Hat OpenShift Pipelines is a cloud-native CI/CD solution for building pipelines using Tekton concepts which run natively on OpenShift and Kubernetes.
     operators.operatorframework.io/builder: operator-sdk-v1.7.2
     features.operators.openshift.io/disconnected: "true"
-    features.operators.openshift.io/fips-compliant: "false"
+    features.operators.openshift.io/fips-compliant: "true"
```
Comment (savitaashture) [inline operatorhub/openshift/manifests/bases/openshift-pipelines-operator-rh.clusterserviceversion.template.yaml:14]: can you point to any doc to read about this annotation

Comment (khrm) [reply operatorhub/openshift/manifests/bases/openshift-pipelines-operator-rh.clusterserviceversion.template.yaml:14]: Please check here: https://docs.openshift.com/dedicated/operators/operator_sdk/osdk-generating-csvs.html#osdk-csv-annotations-infra_osdk-generating-csvs

**Review by savitaashture — APPROVED** [review]

**Review by vdemeester — APPROVED** [review]
/lgtm

**Pull request merged**